src/scripts/contraception/scenarios/run_analysis_co_health_impacts_no_diseases.py

```diff
@@ -0,0 +1,124 @@
+"""
+This file defines a batch run to get sims results to be used by an analysis script
+( # TODO: create one and add its name here
+).
+Run on the remote batch system using:
+```tlo batch-submit src/scripts/contraception/scenarios/run_analysis_co_health_impacts_no_diseases.py```
+or locally using:
+```tlo scenario-run src/scripts/contraception/scenarios/run_analysis_co_health_impacts_no_diseases.py```
+
+# TODO: update
+SCENARIO SETTINGS
+-----------------
+used modules:
+...
+
+# TODO: update
+FP INTERVENTIONS' IMPLICATIONS FOR MNCH PAPER (Eva J et al. 2024):
+------------------------------------------------------------------
+...
+"""
+
+from tlo import Date, logging
+from tlo.methods import (
+    alri,
+    cardio_metabolic_disorders,
+    care_of_women_during_pregnancy,
+    contraception,
+    demography,
+    depression,
+    diarrhoea,
+    enhanced_lifestyle,
+    epi,
+    healthburden,
+    healthseekingbehaviour,
+    healthsystem,
+    hiv,
+    labour,
+    malaria,
+    newborn_outcomes,
+    postnatal_supervisor,
+    pregnancy_supervisor,
+    stunting,
+    symptommanager,
+    tb,
+    wasting,
+)
+from tlo.scenario import BaseScenario
+
+
+class RunAnalysisCo(BaseScenario):
+    def __init__(self):
+        super().__init__(
+            seed=0,
+            start_date=Date(2010, 1, 1),
+            end_date=Date(2030, 12, 31),
+            initial_population_size=2_000,  # selected size for the Tim C at al. 2023 paper: 250K
+            number_of_draws=1,  # <- one scenario
+            runs_per_draw=1,  # <- repeated this many times
+        )
+
+    def log_configuration(self):
+        return {
+            'filename': 'run_analysis_co_health_impacts_no_diseases',  # <- (specified only for local running)
+            'directory': './outputs',  # <- (specified only for local running)
+            'custom_levels': {
+                '*': logging.WARNING,
+                "tlo.methods.demography": logging.INFO,
+                "tlo.methods.demography.detail": logging.INFO,
+                "tlo.methods.depression": logging.INFO,
+                "tlo.methods.contraception": logging.INFO,
+                "tlo.methods.healthsystem.summary": logging.INFO,
+                "tlo.methods.healthburden": logging.INFO,
+                "tlo.methods.hiv": logging.INFO,
+                "tlo.methods.labour": logging.INFO,
+                "tlo.methods.labour.detail": logging.INFO,
+                "tlo.methods.malaria": logging.INFO,
+                "tlo.methods.newborn_outcomes": logging.INFO,
+                "tlo.methods.care_of_women_during_pregnancy": logging.INFO,
+                "tlo.methods.pregnancy_supervisor": logging.INFO,
+                "tlo.methods.postnatal_supervisor": logging.INFO,
+                "tlo.methods.tb": logging.INFO,
+            }
+        }
+
+    def modules(self):
+        return [
```

Comment:
Why not use fullmodel ?

Reply:
I think @joehcollins  mentioned that we need pretty large population to be able to see smt, so at least for preliminary results it didn't make sense to use fullmodel, is that correct @joehcollins?

Reply:
i think this was a direct copy from the scripts i was using for my PhD - we didnt use the full model then because of the run time. I think going forward with this analysis we would want to use the full model